### PR TITLE
fix the download-url of gmock

### PR DESCRIPTION
**Pull request by simonkuang**

### CODE REVIEW

**Comment by simonkuang** [comment]
I signed it!

File: Formula/google-protobuf.rb [hunk 1]
```diff
@@ -59,6 +65,7 @@ def install
     ENV.universal_binary if build.universal?
     ENV.cxx11 if build.cxx11?
 
+    (buildpath/"gmock").install resource("gmock")
```
Comment (stanley-cheung) [inline Formula/google-protobuf.rb:68]: Can you point me to some documentation on how this works? I am not too familiar with this particular syntax. How does `.install` seem to know to unzip the zip file?

Comment (simonkuang) [reply Formula/google-protobuf.rb:68]: @stanley-cheung I have no idea how exactly the `.install` comes from, but the trigger of unzip action comes from [the Resource class](https://github.com/Homebrew/brew/blob/4b2c4ef25835a87b5bac4fb26eae91dbfed93863/Library/Homebrew/resource.rb#L93). And, there is [another `.install` call like this](https://github.com/Homebrew/brew/blob/4b2c4ef25835a87b5bac4fb26eae91dbfed93863/Library/Homebrew/resource.rb#L101).

Comment (vladshub) [reply Formula/google-protobuf.rb:68]: @stanley-cheung sorry for the intrusion, the `.install` function is coming out of an overide that homebrew did on [Pathname.rb](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/extend/pathname.rb#L64) basicly they are creating the destanation path and moving/symlinking there the file that you wish to install.

**Comment by stanley-cheung** [comment]
I will merge this for now. We might have to revert this if it breaks something else, and/or when protobuf comes up with a new released version.

**Pull request merged**